dot/sync/mocks/Network.go

```diff
@@ -59,3 +56,8 @@ func (_m *Network) Peers() []common.PeerInfo {
 
 	return r0
 }
+
+// ReportPeer provides a mock function with given fields: change, p
+func (_m *Network) ReportPeer(change peerset.ReputationChange, p peer.ID) {
+	_m.Called(change, p)
+}
```

Comment:
I think this is not generated, please use `go generate ./...` from the root of the project to re-generate and update mocks.

Reply:
after we changed the interfaces we call `go generate -run "mockery" ./...`, is there something missing?

Reply:
Is this generated with `go generate`? It looks strange there is a comment on this method and not the other ones :thinking: But maybe that's just mockery acting odd as well

Reply:
the comment appears in all generated mocked methods, looks like a default comment changing just the function and params name

Reply:
Oh yeah my apologies. It's just the previous `ReportPeer` didn't have a comment so I thought it was manually added. It's also strange it adds it at a different place when re-generating... Unpredictable mockery :smile_cat:

Reply:
At least I wasn't doing something wrong, thanks guys

Reply:
I opened https://github.com/ChainSafe/gossamer/pull/2062 to have this mock generation check in the CI to avoid any confusion in the future :wink: